### PR TITLE
Add androidicons@1.0 w/ git auto-update

### LINKED ISSUE #9218: [Request] Add androidicons
**Library name:** androidicons
**Git repository url:** https://github.com/opoloo/androidicons

### PR DESCRIPTION
**Pull request by sashberd**
close #9218, cc @thej, using web-minify-helper for minifing files

Pull request for issue: #9218 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/opoloo/androidicons
 * Official website (optional, not the repository): http://www.androidicons.com/
 * NPM package url (optional):
 * License and its reference: CC-BY-SA-4.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 67
   - Count of stars: 672
   - Count of forks: 119
 * NPM download stats (optional):
   - Downloads in the last day: 
   - Downloads in the last week: 
   - Downloads in the last month: 

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/androidicons/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/opoloo/androidicons.git",
+    "fileMap": [
+      {
+        "basePath": "iconfont",
+        "files": [
+          "css/*",
+          "font/*"
```
Comment (sufuf3) [inline ajax/libs/androidicons/package.json]: Let's use `+(css|font)/*`
As usual, we'd like to reduce the line of the pattern, and to get all the files we want.
Thanks.

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Comment by sashberd** [comment]
@sufuf3 ping

**Comment by extend1994** [comment]
@PeterDaveHello I can't merge this PR, please help review & merge it, thanks.
The message shows that "This pull request can be automatically merged by project collaborators".

**Comment by PeterDaveHello** [comment]
@extend1994 hmmm, fixed now, try again?

**Comment by extend1994** [comment]
@PeterDaveHello Fixed now, thank you!

**Pull request merged**